### ISSUE TITLE
[Feature] customize reactions with gif/png/ico files

### ISSUE THREAD

**Issue by c0ncentus** [charkour/react-reactions#1]
hi, thanks to continue the job, make react-reactions great again :) !

**Comment by charkour** [comment]
Hey @c0ncentus,

Which selector or counter are you hoping to customize? I plan to make the Facebook selector and counter customizable (after I finish porting it to Typescript. Currently about halfway done). Thanks for reaching out.

**Comment by c0ncentus** [comment]
Facebook Selector is the most I think 👍 . this will be awesome with GightubCounter et GithubSelector.
Thanks for planning to support this awesome package.


awesome script + awesome feature = awesome UI and awesome package ❤️ .

**Comment by charkour** [comment]
Sounds good! I'm almost done with the Typescript port and then I'll plan on adding custom support for the Facebook Selector and Counter first. 

Thanks for your excitement!

**Comment by crieggalder** [comment]
@charkour glad to see this package getting revived! Thank you for your work on this. :)

**Comment by charkour** [comment]
@c0ncentus, I'm trying to design a way to add custom images to the selectors. I think creating a custom selector component would be best. 

Do you have suggestions on how to implement it? 

Sorry this feature is taking a long time.

**Comment by charkour** [comment]
@c0ncentus, check out PR #62. 

This add's the ability to use custom reactions on the reaction bar

![image](https://user-images.githubusercontent.com/33156025/111041592-e1ece100-8406-11eb-82f5-226b3839683c.png)

Also works with images. 

![image](https://user-images.githubusercontent.com/33156025/111041788-00071100-8408-11eb-82a3-e23723e0755c.png)

Useage: 
### Reaction Bar Selector

```tsx
import React from 'react';
import { ReactionBarSelector } from '@charkour/react-reactions';

const Component = () => {
  return <ReactionBarSelector />;
};
```

**Props:**

`iconSize?: number` — String icon pixel size. Defaults to `38px`
`reactions?: Reaction[];` — Array of Reaction objects `{label: "haha", node: <div>😄</div>}` to display.
`onSelect: (label: string) => void;` — Function callback when emoji is selected

_Note_: When passing an `<img>` as a Reaction. Specify the `iconSize` as the height of the image. `<img height={iconSize} src="img-source" />`

**Comment by charkour** [comment]
This feature is live on [`0.5.0`](https://www.npmjs.com/package/@charkour/react-reactions/v/0.5.0). Let me know what you think and I'll work on adding more custom components.

**Comment by charkour** [comment]
#118 adds the custom reaction counter. It is released in [v0.6.0](https://www.npmjs.com/package/@charkour/react-reactions/v/0.6.0). Thanks to @justemoilouise!
